packages/abi-typegen/src/templates/utils/formatEnums.ts

```diff
@@ -1,22 +1,35 @@
-import type { EnumType } from '../../abi/types/EnumType';
+import type { EnumType, RawTypeHash } from '../../abi/types/EnumType';
 import { TargetEnum } from '../../types/enums/TargetEnum';
 import type { IType } from '../../types/interfaces/IType';
 
 export function formatEnums(params: { types: IType[] }) {
   const { types } = params;
 
+  const typeHash: RawTypeHash = types.reduce(
+    (hash, row) => ({
+      ...hash,
+      [row.rawAbiType.typeId]: row,
+    }),
+    {}
+  );
+
   const enums = types
     .filter((t) => t.name === 'enum')
     .map((t) => {
       const et = t as EnumType; // only enums here
       const structName = et.getStructName();
       const inputValues = et.getStructContents({ types, target: TargetEnum.INPUT });
       const outputValues = et.getStructContents({ types, target: TargetEnum.OUTPUT });
+      const inputNativeValues = et.getNativeEnum(typeHash);
+      const outputNativeValues = et.getNativeEnum(typeHash);
```

Comment:
Please, preserve a similar signature to these:

```suggestion
      const inputNativeValues = et.getNativeEnum({ types });
      const outputNativeValues = et.getNativeEnum({ types });
```